### PR TITLE
add fixed accordion to project page

### PR DESCRIPTION
**Pull request by sarahframe**

### CODE REVIEW

File: cdhweb/pages/templatetags/custom_tags.py [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
 register = template.Library()
 
+
 @register.simple_tag(takes_context=True)
 def unique_id(context, prefix="id-"):
     return f"{prefix}{uuid.uuid4().hex}"
+
+
+@register.simple_tag
+def get_accordion_heading_display(choice_value):
```
Comment (haydngreatnews) [inline cdhweb/pages/templatetags/custom_tags.py:16]: It is _wild_ to me that we have to do this, but it does seem to be the case :(

**Review by haydngreatnews — APPROVED** [review]

File: cdhweb/pages/templatetags/custom_tags.py (hunk 1 above)
Comment (liamjohnston) [reply cdhweb/pages/templatetags/custom_tags.py:16]: So what's going on here? To clarify, we don't want to give the CMS user a choice (if that's what's happening). 
These accordions are basically hard-coded as the first content on the page (after the hero). There's no `h2` above them, therefore the heading level for these should be `h2`.
Screenshot 2024-07-04 at 8 54 40 AM

File: cdhweb/projects/migrations/0027_project_accordion_alter_project_description_and_more.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                                                        choices=[
+                                                            ("code", "Code"),
+                                                            ("data", "Data"),
+                                                            (
+                                                                "documentation",
+                                                                "Documentation",
+                                                            ),
+                                                            (
+                                                                "related_courses_and_course_modules",
+                                                                "Related Courses and Course Modules",
```
Comment (liamjohnston) [inline cdhweb/projects/migrations/0027_project_accordion_alter_project_description_and_more.py:40]: ~What is this? Sounds Massey-ish 🤔~

Comment (liamjohnston) [reply cdhweb/projects/migrations/0027_project_accordion_alter_project_description_and_more.py:40]: wait nm, i see it in figma now! 😆

File: cdhweb/pages/templatetags/custom_tags.py (hunk 1 above)
Comment (liamjohnston) [reply cdhweb/pages/templatetags/custom_tags.py:16]: I was thinking this was to do with heading _levels_, not the heading text. Ignore me, once again.

**Pull request merged**